validate_depctrl.py

```diff
@@ -59,15 +63,22 @@ def check_changelog(lines, changelog, filename):
 
 def check_contents(entry, basename):
     all_fine = True
-    for ch in entry['channels'].values():
-        for f in ch['files']:
-            filename = basename + f['name']
-            with open(filename, 'rb') as fo:
+    for ch in entry["channels"].values():
+        for f in ch["files"]:
+            filename = basename + f["name"]
+            with open(filename, "rb") as fo:
```

Comment:
i prefer to single-quote "identifiers" (like dict keys), enum-like arguments, and so on, only double-quoting "real strings"
i.e. these all would be left as `'` by that style (as well as `'utf-8'` below), while `replace("\r", "").split("\n")` would stay as double-quoted (and, yes, `"utf-8"` was originally mine; i forget sometimes)

much as many in this thread: https://stackoverflow.com/a/56190